crates/uv/tests/pip_compile.rs

```diff
@@ -6458,32 +6460,32 @@ fn no_strip_markers_multiple_markers() -> Result<()> {
     "})?;
 
     uv_snapshot!(context.filters(), context.pip_compile()
-            .arg("requirements.in")
-            .arg("--no-strip-markers")
-            .arg("--python-platform")
-            .arg("windows"), @r###"
+        .arg("requirements.in")
+        .arg("--no-strip-markers")
+        .arg("--python-platform")
+        .arg("windows"), @r###"
     success: true
     exit_code: 0
     ----- stdout -----
     # This file was autogenerated by uv via the following command:
     #    uv pip compile --cache-dir [CACHE_DIR] requirements.in --no-strip-markers --python-platform windows
-    attrs==23.2.0 ; sys_platform == 'win32' or python_full_version >= '3.12'
+    attrs==23.2.0
         # via
         #   outcome
         #   trio
-    cffi==1.16.0 ; (implementation_name != 'pypy' and os_name == 'nt' and sys_platform == 'win32') or (python_full_version >= '3.12' and implementation_name != 'pypy' and os_name == 'nt')
+    cffi==1.16.0 ; implementation_name != 'pypy' and os_name == 'nt'
         # via trio
-    idna==3.6 ; sys_platform == 'win32' or python_full_version >= '3.12'
+    idna==3.6
         # via trio
-    outcome==1.3.0.post0 ; sys_platform == 'win32' or python_full_version >= '3.12'
+    outcome==1.3.0.post0
         # via trio
-    pycparser==2.21 ; (implementation_name != 'pypy' and os_name == 'nt' and sys_platform == 'win32') or (python_full_version >= '3.12' and implementation_name != 'pypy' and os_name == 'nt')
+    pycparser==2.21 ; implementation_name != 'pypy' and os_name == 'nt'
         # via cffi
-    sniffio==1.3.1 ; sys_platform == 'win32' or python_full_version >= '3.12'
+    sniffio==1.3.1
         # via trio
-    sortedcontainers==2.4.0 ; sys_platform == 'win32' or python_full_version >= '3.12'
+    sortedcontainers==2.4.0
         # via trio
-    trio==0.25.0 ; sys_platform == 'win32' or python_full_version >= '3.12'
```

Comment:
The simplification of markers here is somewhat debatable. I might rather _not_ do this after Andrew's PR merges, since we don't record a `requires-python` in these files.